Source/WTF/wtf/Forward.h

```diff
@@ -75,14 +75,15 @@ struct VectorBufferMalloc;
 #else
 using VectorBufferMalloc = FastMalloc;
 #endif
+using EmbeddedFixedVectorMalloc = FastMalloc;
```

Comment:
This is not correct.  In EmbeddedFixedVector.h, `DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(EmbeddedFixedVector);` defines `EmbeddedFixedVectorMalloc`.  It is not always `FastMalloc`.

See how `VectorBufferMalloc` is handled above for the treatment you should apply.

Reply:
Fixed